### PR TITLE
Rename all tests so they are discoverable by pytest

### PR DESCRIPTION
**Pull request by memsharded**
Changelog: omit
Docs: omit

#tags: slow, svn

### CODE REVIEW

File: conans/test/functional/command/alias_test.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def requirements( self):
         self.assertIn('"CK/0.1@user/testing" -> "CH/0.1@user/testing"', graphfile)
 
     @parameterized.expand([(True, ), (False, )])
-    def test_double_alias_test(self, use_requires):
+    def test_test_double_alias(self, use_requires):
```
Comment (jgsogo) [inline conans/test/functional/command/alias_test.py]:
```suggestion
    def test_double_alias(self, use_requires):
```

File: conans/test/functional/command/alias_test.py [hunk 2]
```diff
@@ -392,13 +392,13 @@ def transitive_alias_test(self):
...
 
-    def repeated_alias_test(self):
+    def test_repeated_alias(self):
         client = TestClient()
         client.run("alias Hello/0.X@lasote/channel Hello/0.1@lasote/channel")
         client.run("alias Hello/0.X@lasote/channel Hello/0.2@lasote/channel")
         client.run("alias Hello/0.X@lasote/channel Hello/0.3@lasote/channel")
 
-    def test_basic_test(self):
+    def test_test_basic(self):
```
Comment (jgsogo) [inline conans/test/functional/command/alias_test.py]:
```suggestion
    def test_basic(self):
```

File: conans/test/functional/command/config_install_test.py [hunk 3]
```diff
@@ -205,7 +205,7 @@ def build(self):
         self.client.run("create . Pkg/0.1@user/testing")
         self.assertIn("A is 3", self.client.out)
 
-    def test_install_file_test(self):
+    def test_test_install_file(self):
```
Comment (jgsogo) [inline conans/test/functional/command/config_install_test.py]:
```suggestion
    def test_install_file(self):
```

File: conans/test/functional/command/config_install_test.py [hunk 4]
```diff
@@ -214,7 +214,7 @@ def test_install_file_test(self):
             self._check("file, %s, True, None" % zippath)
             self.assertTrue(os.path.exists(zippath))
 
-    def test_install_config_file_test(self):
+    def test_test_install_config_file(self):
```
Comment (jgsogo) [inline conans/test/functional/command/config_install_test.py]:
```suggestion
    def test_install_config_file(self):
```

File: conans/test/functional/command/config_install_test.py [hunk 5]
```diff
@@ -239,7 +239,7 @@ def test_install_config_file_test(self):
             self.client.run('config install "%s" %s' % (src_remote_file, cmd_option))
             self._check("file, %s, True, None" % src_remote_file)
 
-    def test_install_dir_test(self):
+    def test_test_install_dir(self):
```
Comment (jgsogo) [inline conans/test/functional/command/config_install_test.py]:
```suggestion
    def test_install_dir(self):
```

File: conans/test/functional/command/create_test.py [hunk 6]
```diff
@@ -400,7 +400,7 @@ def build_policy_test(self):
         client.run("create . pkg/0.1@user/stable")
         self.assertIn("Bar/0.1@user/stable: Forced build from source", client.out)
 
-    def test_build_folder_handling_test(self):
+    def test_test_build_folder_handling(self):
```
Comment (jgsogo) [inline conans/test/functional/command/create_test.py]:
```suggestion
    def test_build_folder_handling(self):
```

File: conans/test/functional/command/inspect_test.py [hunk 7]
```diff
@@ -489,7 +489,7 @@ class Lib(ConanFile):
         client.run("inspect . --raw=default_options")
         self.assertEqual("dict=True\nlist=False", client.out)
 
-    def test_initial_inspect_without_registry_test(self):
+    def test_test_initial_inspect_without_registry(self):
```
Comment (jgsogo) [inline conans/test/functional/command/inspect_test.py]:
```suggestion
    def test_initial_inspect_without_registry(self):
```

File: conans/test/functional/configuration/conan_trace_file_test.py [hunk 8]
```diff
@@ -19,7 +19,7 @@ def setUp(self):
         test_server = TestServer()
         self.servers = {"default": test_server}
 
-    def test_run_log_file_package_test(self):
+    def test_test_run_log_file_package(self):
```
Comment (jgsogo) [inline conans/test/functional/configuration/conan_trace_file_test.py]:
```suggestion
    def test_run_log_file_packaged(self):
```

**Review by jgsogo — APPROVED** [review]
Very boring PR to review 😅 

I think I spotted some tests whose names can be simplified because of duplicated `test_test_` (and not referring to _test_ command or _test package_).

But, we can merge.

**Comment by uilianries** [comment]
Great! Now I can run pytest!

**Comment by memsharded** [comment]
We are moving to pytest also for the new testing framework, and also CI will run pytest when this happens.

**Comment by uilianries** [comment]
Great, so you can consider https://github.com/conan-io/conan/issues/6527

**Comment by memsharded** [comment]
Sure, moved #6527 to "in-progress". Lets use that as the main tracker of this effort.

**Pull request merged**